src/util.cpp

```diff
@@ -85,6 +85,8 @@ const int64_t nStartupTime = GetTime();
 const char * const BITCOIN_CONF_FILENAME = "bitcoin.conf";
 const char * const BITCOIN_PID_FILENAME = "bitcoind.pid";
 
+fs::detail::utf8_codecvt_facet g_utf8;
```

Comment:
static?

Reply:
`g_utf8` should be accessible globally.

Reply:
Looks like the only use is in util.h, if you implement those as methods in util.cpp, declared in util.h, you can limit variable access to here util.cpp. I may be wrong though! Reviewing on a cellphone. :P